NEWS.md

```diff
@@ -1,5 +1,25 @@
 ## 8.0.0 - Unreleased
 
+## 7.1.0 - Released
```

Comment:
I believe the release version has to be major i.e. `8.0.0` because of braking changes like removal of `orders.acquisitions-unit-assignments` interface and endpoints: [v7.0.1 vs. master](https://github.com/folio-org/mod-orders/compare/v7.0.1...master)